src/main/scala/io/keen/client/scala/Client.scala

```diff
@@ -15,13 +17,14 @@ class Client(
   ) extends HttpAdapterComponent with Logging {
 
   val httpAdapter: HttpAdapter = new HttpAdapterSpray
+  val settings: Settings = new Settings(config)
 
   /**
    * Disconnects any remaining connections. Both idle and active. If you are accessing
    * Keen through a proxy that keeps connections alive this is useful.
    */
-  def shutdown {
-    httpAdapter.shutdown
+  def shutdown() = {
```

Comment:
Using parens when arity-0 methods are purely side effect with `Unit` return [is a strongly encouraged style convention][1] to make them stand out, so I've tried to apply that consistently among these changes. It's necessary to put them at definition site, or else it actually becomes an error to use them at call site.

[1]: http://docs.scala-lang.org/style/method-invocation.html#arity-0